### PR TITLE
choice authorizers are informees

### PR DESCRIPTION
**Pull request by andreaslochbihler-da**

### CODE REVIEW

**Review by nickchapman-da — APPROVED** [review]
LGTM

But where is `protoActionNodeInfo`  used?

**Comment by andreaslochbihler-da** [comment]
> But where is protoActionNodeInfo used?

I have no idea.

File: daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/TransactionCoder.scala [hunk 1]
```diff
@@ -893,14 +893,20 @@ object TransactionCoder {
               Right(Set.empty[Party])
             else
               toPartySet(protoExe.getObserversList)
+          choiceAuthorizers_ <-
+            if (txVersion < TransactionVersion.minChoiceAuthorizers) {
```
[the remark this answers is not in the document]
Comment (andreaslochbihler-da) [reply daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/TransactionCoder.scala:897]: This should already be tested by `com.daml.lf.transaction.TransactionCoderSpec."do Node.Exercise"`, shouldn't it?

**Pull request merged**